PORTAL/jobs/_utils.py

```diff
@@ -1085,8 +1085,6 @@ def from_raw(cls, raw, *, name=None):
     def __init__(self, root):
         if not root or root == '.':
             root = CWD
-        else:
-            root = os.path.abspath(os.path.expanduser(root))
```

Comment:
This would do tilde expansion on the portal machine that wouldn't be correct for the benchmarking machine.  We could do this for paths we know only exist on the portal machine, but at this level (deep in this base class) it could be either.

Reply:
Got it.  Thanks for explaining.  How about we give the option to not expand?

```python
    def __init__(self, root, *, resolve=True):
        if not root or root == '.':
            root = CWD
        elif resolve:
            root = os.path.abspath(os.path.expanduser(root))
        super().__init__(root=root)
```

Then `WorkerJobsFS`, etc. can do the right thing.

----

...or differentiate remote trees:

```python
    def __init__(self, root, *, remote=False):
        if not root or root == '.':
            root = CWD
        elif not remote:
            root = os.path.abspath(os.path.expanduser(root))
        super().__init__(root=root)
```

...or tell `FSTree` the "os":

```python
    def __init__(self, root, *, osname=None):
        if not root or root == '.':
            root = CWD
        elif osname:
            # XXX normalize?
            pass
        else:
            root = os.path.abspath(os.path.expanduser(root))
        super().__init__(root=root)
```

Reply:
It might make sense to punt on this -- I've already started work on a branch to improve a bunch of path-related things to help with Windows support -- this probably makes more sense as part of that.

Reply:
I'm fine with punting.  The normalization of `root` is just a convenient bit of functionality when working locally.  I'd rather not lose it if possible.